packages/core/src/util.ts

```diff
@@ -51,14 +51,63 @@ export function makeBacktrace(stack: string, shift = 0): BacktraceFrame[] {
           column: line.column
         }
       })
-    backtrace.splice(0, shift)
+    if (filterHbSourceCode) {
+      backtrace.splice(0, calculateBacktraceShift(backtrace))
+    }
+
     return backtrace
   } catch (_err) {
     // TODO: log error
     return []
   }
 }
 
+const DEFAULT_BACKTRACE_SHIFT = 3
+
+function isFrameFromHbSourceCode(frame: BacktraceFrame) {
+  return frame.file.indexOf('@honeybadger-io') > -1 || frame.method.indexOf('@honeybadger-io') > -1
```

Comment:
Perhaps more readable as `frame.file.includes('@honeybadger-io') || frame.method.includes('@honeybadger-io')`

Also can we assume lower case?

Reply:
I usually go for the `indexOf` because it has better browser support, but I guess this shouldn't be a concern anymore. I will change it!

> Also can we assume lower case?

I think so. But it doesn't hurt to add a `toLowerCase()`! Will do!